scripts/upload_zoom_recordings.py

```diff
@@ -68,6 +69,7 @@ def download_file(url, download_path):
                     command = [
```

Comment:
Don't know if in an issue, but from the file upload manager we were setting the license as Creative Commons Attribution license (reuse allowed)... can we set that with `youtube-upload`?